plugins/gcloud/lib/samson_gcloud/image_scanner.rb

```diff
@@ -5,8 +5,9 @@ class ImageScanner
     SUCCESS = 1
     FOUND = 2
     ERROR = 3
+    UNSUPPORTED = 4
 
-    FINISHED = [SUCCESS, FOUND].freeze
+    FINISHED = [SUCCESS, FOUND, UNSUPPORTED].freeze
```

Comment:
ERROR is also FINISHED, no?

Reply:
or we do a retry?

Reply:
retry since it could be a temporary error